code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -76,19 +167,44 @@
 				/obj/item/seeds/sunflower/moonflower = 8
 				)
 
-/obj/effect/mob_spawn/human/seed_vault
-	name = "vault creature sleeper"
-	mob_name = "Vault Creature"
-	icon = 'icons/obj/Cryogenic2.dmi'
-	icon_state = "sleeper"
-	roundstart = FALSE
-	death = FALSE
-	mob_species = /datum/species/pod
-	flavour_text = {"You are a strange, artificial creature. Your creators were a highly advanced and benevolent race, and launched many seed vaults into the stars, hoping to aid fledgling civilizations. You are to tend to the vault and await the arrival of sentient species. You've been waiting quite a while though..."}
+//Ash walker nest
+/mob/living/simple_animal/hostile/spawner/ash_walker
+	name = "ash walker nest"
+	desc = "A nest built around a necropolis tendril. The eggs seem to grow unnaturally fast..."
+	icon = 'icons/mob/nest.dmi'
+	icon_state = "ash_walker_nest"
+	icon_living = "ash_walker_nest"
+	faction = list("ashwalker")
+	health = 200
+	maxHealth = 200
+	loot = list(/obj/effect/gibspawner, /obj/item/device/assembly/signaler/anomaly)
+	del_on_death = 1
+	var/meat_counter
 
-/obj/effect/mob_spawn/human/seed_vault/special(mob/living/new_spawn)
-	var/plant_name = pick("Tomato", "Potato", "Brocolli", "Carrot", "Deathcap", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Bannana", "Moss", "Flower", "Bloom", "Spore", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocao", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
-	new_spawn.real_name = plant_name
+/mob/living/simple_animal/hostile/spawner/ash_walker/Life()
+	..()
+	if(!stat)
+		consume()
+		spawn_mob()
+
+/mob/living/simple_animal/hostile/spawner/ash_walker/proc/consume()
+	for(var/mob/living/H in view(src,1)) //Only for corpse right next to/on same tile
+		if(H.stat)
```

Comment:
I figured that unconsciousness worked to consume them, as it's been a regular tactic of ash walkers to feed critted brethren to the tendril